test/iron-overlay-behavior.html

```diff
@@ -421,6 +434,45 @@
 
       });
 
+      suite('multiple overlays with backdrop implemented in different elements', function () {
+        var overlays;
+
+        setup(function() {
+          overlays = fixture('backdrop-different-elements');
+        });
+
+        test('multiple overlays share the same backdrop', function() {
+          assert.equal(overlays[0].backdropElement, overlays[1].backdropElement);
+        });
+
+        test('multiple overlays close the corrent backdrop', function(done) {
+          runAfterOpen(overlays[0], function () {
```

Comment:
Consider adding a method `runAfterClose` which is the same as `runAfterOpen`, but just adds a listener for `iron-overlay-close` instead of `iron-overlay-opened`.
Also, notice that `runAfterOpen` will execute the callback after 1ms.
With that this code could become something like this:
```javascript
runAfterOpen(overlays[0], function() {
  overlays[0].close();
  runAfterOpen(overlays[1], function() {
    runAfterClose(overlays[1], function() {
       // your tests here!
    });
  });
});